examples/experimental/dagster-airlift/dagster_airlift/core/multi_asset.py

```diff
@@ -10,14 +10,16 @@
 class PythonDefs(DefsFactory):
     specs: List[AssetSpec]
     name: str
+    group: Optional[str] = None
     python_fn: Optional[Callable] = None
 
     def build_defs(self) -> Definitions:
         @multi_asset(
             specs=self.specs,
             name=self.name,
+            group_name=self.group,
         )
-        def _multi_asset() -> None:
```

Comment:
why this change?

Reply:
An actually executable multi asset can't have a `None` annotation. Throws an error when executing, it expects a tuple (one for each spec I'd assume or something)